scss/titles.scss

```diff
@@ -0,0 +1,14 @@
+.page-title {
+  position: relative;
+  left: 70px;
+  top: 50px;
+  &::before {
+    content: url('assets/triangle.svg');
+    height: 90px;
+    width: 70px;
```

Comment:
Normally you only need height OR width for images. Try it and if it's not the same you can ignore this comment